client/src/components/ReviewTab/ReviewTab.tsx

```diff
@@ -38,7 +72,36 @@ export default function PersonalInfo(): JSX.Element {
             <ReviewTabForm handleSubmit={handleSubmit} />
           </Grid>
         </Grid>
-      </Grid>
+      </Paper>
+      <Typography align="center" component="h1" variant="subtitle1" className={classes.readAll}>
+        Read all
+      </Typography>
+      <Switch checked={checked} onChange={handleChange} inputProps={{ 'aria-label': 'controlled' }} />
+      {checked === true
+        ? reviews &&
```

Comment:
We may not need the short circuit operator as we are mapping over the reviews array here

Reply:
handled that in the recent PR for artist page UI improvement